network/src/main/java/com/arcadedb/remote/RemoteDatabase.java

```diff
@@ -400,7 +400,7 @@ public ResultSet command(final String language, final String command, final Cont
     checkDatabaseIsOpen();
     stats.commands.incrementAndGet();
 
-    return (ResultSet) databaseCommand("command", language, command, params, true,
```

Comment:
Why are you setting false = not require a leader? This means the operation will go to any server, but if it's an update (query is idempotent) will bounce to the leader in 2 passes